trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'baseDecorator'
```

Comment:
- [ ] It would be better if we address the path, currently we are unable to test if the decorators are working as expected. Let us replace `require_relative 'baseDecorator'` with `require_relative './base_decorator'`